introduction.md

```diff
@@ -1 +1,5 @@
 hello, i am taking the course 'Mastering GITHUB', my favorite Code School path so far is JavaScript...
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?